README.md

```diff
@@ -55,3 +55,21 @@ Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser
 
 ```git remote -v ```
 tells us / lists our remotes
+
+
+### See below for how to build the two docker images 
+```
+docker build --target development --tag todo-app:dev .
+docker build --target production --tag todo-app:prod .
+```
+### Command to run the production container
+``` 
+docker run --env-file .env -it -p 5001:5000 todo-app:prod
+```
+This command pases through the environment variables, `-it` makes it easier to interact with the container, the app can be accessed at port `http://localhost:5001`  
```

Comment:
I know we wrote this at speed together, but potentially we could make this a little clearer for future readers. Like code, documentation will usually be read many more times than it is written, so it's worth optimising for that future clarity!
```suggestion
This command pases through the environment variables with the `--env-file` flag, and the `-it` flags make it easier to interact with the container (e.g. allowing us to shut it down with ctrl+c from our host terminal). With the `-p` flag, the app can be accessed at the address `http://localhost:5001`.
```